docs/how-to-guides/themes/theme-support.md

```diff
@@ -476,3 +476,17 @@ Use this setting to enable the following Global Styles settings:
 ```php
 add_theme_support( 'appearance-tools' );
 ```
+
+## Block Based Template Parts
+
+Block Based Template parts allow editors to edit parts of the site using blocks. This is off by default, and requires the theme to opt in by declaring support:
```

Comment:
The `editors` as in role Editor? That would be false since only the site admins can access the Appearance menu.

Reply:
Good call. I thought of it as editor as in the person. But that is confusing with the role. I will update it to admin